main/settings.py

```diff
@@ -120,6 +121,9 @@
         'PASSWORD': "osp",
         'HOST': "localhost", # Change to db for docker-compose
         'PORT': 5432,
+        'TEST':{
```

Comment:
this thing is not needed.

Reply:
Owh , It will create test database on it's own . 
Btw what will be the name of the database created?
Will it be same "test_osp" ? And if it can create test database on it's own , then what is the need of this Test {name: ""}

Reply:
Yes, the alias database name will be `test_osp` and the owner will be the user `osp`.
There is no need to specify 'TEST' in database configuration, you can remove these lines. @decon-hars

Reply:
Resolved